packages/redux/src/themes/selectors.ts

```diff
@@ -0,0 +1,49 @@
+import { getError, getIsLoading, getResult } from './reducer.js';
+import type { StoreState } from '../types/index.js';
+import type { Themes } from '@farfetch/blackout-client';
+import type { ThemesState } from './types/index.js';
+
+/**
+ * Checks if styleguide themes has error.
+ *
+ * @param state - Application state.
+ * @param code  - The theme identifier.
+ *
+ * @returns Staff member error.
+ */
+export const getThemesError = (state: StoreState, code: Themes['code']) =>
+  getError(state.themes as ThemesState)?.[code];
+
+/**
+ * Checks if styleguide themes is loading.
+ *
+ * @param state - Application state.
+ * @param code  - The theme identifier.
+ *
+ * @returns If the staff member is loading or not.
+ */
+export const isThemesLoading = (state: StoreState, code: Themes['code']) =>
```

Comment:
Either `areThemesLoading` or `isThemeLoading` (if you pick up on my previous comment about having this named in singular) 😄

Reply:
Done